swing/src/net/sf/openrocket/gui/configdialog/AppearancePanel.java

```diff
@@ -463,6 +465,8 @@ public void actionPerformed(ActionEvent e) {
 					c.fireComponentChangeEvent(ComponentChangeEvent.NONFUNCTIONAL_CHANGE);
 					if (outsideInsidePane != null)
 						outsideInsidePane.setEnabledAt(1, !customInside.isSelected());
+					if (edgesCheckbox != null)
```

Comment:
nit: while style previously has no brackets, I think we should *always* use brackets, even for one line if statements. However, this is a style nit for now. In the future, we should add a linter of sorts to check a coding style we can agree upon.